udata/core/user/models.py

```diff
@@ -236,9 +244,37 @@ def mark_as_deleted(self):
         Follow.objects(following=self).delete()
         mail.send(_('Account deletion'), copied_user, 'account_deleted')
 
+    def count_datasets(self):
+        from udata.models import Dataset
+        self.metrics["datasets"] = Dataset.objects(owner=self).visible().count()
+        self.save()
 
-datastore = MongoEngineUserDatastore(db, User, Role)
+    def count_reuses(self):
+        from udata.models import Reuse
+        self.metrics["reuses"] = Reuse.objects(owner=self).count()
+        self.save()
 
+    def count_followers(self):
+        from udata.models import Follow
+        self.metrics["followers"] = Follow.objects.followers(self).count()
+        self.save()
+    
+    def count_following(self):
+        from udata.models import Follow
+        self.metrics["following"] = Follow.objects.following(self).count()
+        self.save()
+    
+    @property
+    def get_metrics(self):
+        return {
```

Comment:
nitpick: since those dict keys are the same than on self.metrics, maybe mutualise a list of keys.

Reply:
cf above